@navikt/core/tokens/src/index.js

```diff
@@ -43,6 +43,7 @@ module.exports = {
       modal: { value: 2000 },
       popover: { value: 1000 },
       focus: { value: 10 },
+      max: { value: 2147483647 },
```

Comment:
Kanskje denne også burde ha et semantisk navn som de andre? Og er det et poeng at den skal være absolutt max?
```suggestion
      tooltip: { value: 3000 },
```

Reply:
Oppdaterte til `navds-z-index-tooltip` 🚀  Tror absolutt max bare er en nice to have her siden man sikrer at tooltip alltid er øverst, men ser ingen problem med å bare bruke en token med eks 3000 her.